src/common/cipher_openssl.c

```diff
@@ -31,29 +31,38 @@ ossl_cipher_ctx_create(void)
 	return EVP_CIPHER_CTX_new();
 }
 
-void
+bool
 ossl_cipher_setup(void)
 {
 #ifdef HAVE_OPENSSL_INIT_CRYPTO
 	/* Setup OpenSSL */
-	OPENSSL_init_crypto(OPENSSL_INIT_LOAD_CONFIG, NULL);
+	if (!OPENSSL_init_crypto(OPENSSL_INIT_LOAD_CONFIG, NULL))
+		return false;
+	return true;
 #endif
+	return false;
```

Comment:
I think we can ignore the return value of `OPENSSL_init_crypto` as we do for `OPENSSL_init_ssl`.